bin/utility/utility.py

```diff
@@ -234,52 +234,58 @@ def check_yangcatalog_data(config: configparser.ConfigParser, yang_file_pseudo_p
     if name_revision in all_modules:
         module_data = all_modules[name_revision].copy()
         update = False
-
-        for field in ['document-name', 'reference', 'author-email']:
-            if new_module_data.get(field) and module_data.get(field) != new_module_data[field]:
-                update = True
-                module_data[field] = new_module_data[field]
-
-        if new_module_data.get('compilation-status') \
-                and module_data.get('compilation-status') != new_module_data['compilation-status'].lower().replace(' ', '-'):
-            # Module parsed with --ietf flag (= RFC) has higher priority
-            if is_rfc and ietf_type is None:
-                pass
-            else:
-                update = True
-                module_data['compilation-status'] = new_module_data['compilation-status'].lower().replace(' ', '-')
-
-        if new_module_data.get('compilation-status') is not None:
-            file_url = _generate_compilation_result_file(module_data, compilation_results,
-                                                         result_html_dir, is_rfc, versions, ietf_type)
-            if module_data.get('compilation-status') == 'unknown':
-                comp_result = ''
-            else:
-                comp_result = '{}/results/{}'.format(domain_prefix, file_url)
-            if module_data.get('compilation-result') != comp_result:
-                update = True
-                module_data['compilation-result'] = comp_result
-
-        if ietf_type is not None and module_data.get('organization') == 'ietf' and 'document-name' in new_module_data:
-            wg = _resolve_working_group(name_revision, ietf_type, new_module_data['document-name'])
-            if (module_data.get('ietf') is None or module_data['ietf']['ietf-wg'] != wg) and wg is not None:
-                update = True
-                module_data['ietf'] = {}
-                module_data['ietf']['ietf-wg'] = wg
-
-        mat_level = _resolve_maturity_level(ietf_type, new_module_data.get('document-name'))
-        if module_data.get('maturity-level') != mat_level:
-            if mat_level == 'not-applicable' and module_data.get('maturity-level'):
-                pass
-            else:
-                update = True
-                module_data['maturity-level'] = mat_level
-
-        if update:
-            updated_modules.append(module_data)
-            print('DEBUG: updated_modules: {}'.format(name_revision))
     else:
         print('WARN: {} not in Redis yet'.format(name_revision))
+        name, revision = name_revision.split('@')
+        module_data: t.Dict[str, t.Any] = {
+            'name': name,
+            'revision': revision,
+            'organization': 'unknown'
```

Comment:
Default value for organization property should be: `independent`

However, organization is necessary for creating compilation result filename -> What will the filename be if we don't have an organization available?

Reply:
There is no way to know the real organization of the module in this case. The organization gets resolved when `populate.py` runs on the backend. If that had already happened, the module would already have been in redis. So it's not possible to create the filename properly, best we can do is give it a temporary name, that is why I used `unknown`.

Reply:
Is it necessary for the organization to be in the filename? Maybe we could just remove it in all cases.

Reply:
Maybe it would be a good idea to check where the given files are used, or where their filenames are used and find out if the organization can be omitted from the name.

Reply:
I've checked, and the filename only ever seems to be saved to the `'compilation-result'` field in redis and  isn't checked in any other places. So removing the organization from the name should be fine.

Reply:
Actually it seems like the filenames as well as some other duplicate data is also resolved when populating modules. That will need to be removed first.

Reply:
So we need to:
- [ ] go through directory and rename each file -> remove organization from name
- [ ] go through all Redis modules and update `compilation-result` property to refer to correct file
anything else?

Reply:
I think that's all that needs to be done after the changes when populating modules.